modules/luigi/phrases/farewell.yml

```diff
@@ -0,0 +1,20 @@
+type: phrases
+name: "payment"
+opts:
+- "alrighty, payment time then! you owe me {cost} roboy coins for the {order}"
+- "Wow it took some time. That will be {cost} roboy coins for the {order} "
```

Comment:
I think we always write everything in lower case... maybe you can adjust this everywhere? :)

Reply:
Sure!